test/mocked_sc_htmlmerger.js

```diff
@@ -14,9 +14,10 @@ connect().use('/sc/htmlmerger', function fooMiddleware(req, res, next) {
         // res.write("Chunk");
         const parts = pair.split('=');
         const appName = parts[0];
-        const fileName = parts[1];
+        let fileName = parts[1];
         content += `<imported-template-scope scope="${appName}"><template><meta itemprop="juicy-composition-scope" content="${appName}"/></template>`;
         if(fileName){
+            fileName = fileName.replace('%2F','/');
```

Comment:
This replaces the first instance only. Are you sure that file name has only one `/`? And I would replace with [`path.sep`](https://nodejs.org/api/path.html#path_path_sep) to cover unix-like and windows.

Reply:
Perhaps use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent?

Reply:
`decodeURIComponent` is a Web API. This file runs within Node.